packages/reviews/judge/src/index.ts

```diff
@@ -9,12 +9,16 @@ type CreateJudgeClientArgs = {
 }
 
 export function createJudgeClient({ baseUrl, apiKey, shopDomain }: CreateJudgeClientArgs) {
+  if (!apiKey || !baseUrl) {
+    throw new Error("Judge me: Invalid keys to create Judge.Me client")
```

Comment:
Seems like it's throwing during the build, maybe console.warn and early return instead?